src/protagonist/API/Features/DeliveryChannels/Requests/UpdateCustomerDefaultDeliveryChannel.cs

```diff
@@ -0,0 +1,90 @@
+﻿using API.Features.DeliveryChannels.Converters;
+using API.Infrastructure.Requests;
+using DLCS.Core;
+using DLCS.HydraModel;
+using DLCS.Repository;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+
+namespace API.Features.DeliveryChannels.Requests;
+
+public class UpdateCustomerDefaultDeliveryChannel : IRequest<ModifyEntityResult<UpdateDefaultDeliveryChannelResult>>
+{
+    public int Customer { get; }
+
+    public int Space;
+    
+    public DefaultDeliveryChannel DefaultDeliveryChannel { get; }
+
+    public UpdateCustomerDefaultDeliveryChannel(int customerId, int space, DefaultDeliveryChannel defaultDeliveryChannel)
+    {
+        Customer = customerId;
+        DefaultDeliveryChannel = defaultDeliveryChannel;
+        Space = space;
+    }
+}
+
+public class UpdateDefaultDeliveryChannelResult
+{
+    public DLCS.Model.DeliveryChannels.DefaultDeliveryChannel? DefaultDeliveryChannel;
```

Comment:
As above - why use a `field` rather than `property`?

Reply:
Think this was missed when I was going back over them - will get it updated